src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -265,6 +266,7 @@ public void storeContext(Context context, HttpServletRequest request) {
 		customValuesMap.put(LIS_PERSON_NAME_FAMILY, request.getParameter(LIS_PERSON_NAME_FAMILY));
 		customValuesMap.put(LIS_PERSON_NAME_GIVEN, request.getParameter(LIS_PERSON_NAME_GIVEN));
 		customValuesMap.put(CUSTOM_CANVAS_USER_ID, request.getParameter(CUSTOM_CANVAS_USER_ID));
+		customValuesMap.put(SESSION_ROLES_FOR_ADDING_TEACHER, appExtPropertiesFile.getProperty(ROLE_CAN_ADD_TEACHER));
```

Comment:
What happens if the property isn't found?  `getProperty()` will return null if it can't find it *anywhere*, but will code that uses the null later do Bad Things™?